src/ReactCompat.ts

```diff
@@ -0,0 +1,12 @@
+import { findDOMNode } from 'react-dom';
```

Comment:
Maybe let's add a comment why this file even exists

Reply:
Done in [f94639c](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/f94639ce662fb81b6696a3a549ba2122c0880207).